Resources/Prototypes/secret_weights.yml

```diff
@@ -1,8 +1,7 @@
 - type: weightedRandom
   id: Secret
   weights:
-    Nukeops: 0.20
-    Traitor: 0.50
+    Nukeops: 0.30
+    Traitor: 0.60
     Zombie: 0.10
-    Revolutionary: 0.20
```

Comment:
How to tell someone doesn't like zombie.

Reply:
good

Reply:
add peashooters then we raise the probability